<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeedyForm</class>
 <widget class="QDialog" name="SpeedyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Request deletion of a page</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../huggle_res/pictures.qrc">
    <normaloff>:/huggle/pictures/Resources/huggle3_small.ico</normaloff>:/huggle/pictures/Resources/huggle3_small.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbInfo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbReason">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Reason:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbReason"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbParameter">
       <property name="text">
        <string>Parameter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leParameter"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbSendWarning">
     <property name="text">
      <string>Send a warning template to user</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTag">
       <property name="text">
        <string>Request</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../huggle_res/pictures.qrc"/>
 </resources>
 <connections/>
</ui>
